.changeset/salty-teams-follow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/connectivity': patch
+---
+
+[Fixed Issue] Add `proxyConfiguration` on the fly to avoid expired proxy authorization token in cached destination if it lives shorter than the token for destination service.
```

Comment:
not sure if we need a changeset for this in v4

Reply:
The thing is that if no changeset, our check-pr workflow will fail..